runtime/src/lib.rs

```diff
@@ -29,22 +29,25 @@
 include!(concat!(env!("OUT_DIR"), "/wasm_binary.rs"));
 
 use fp_rpc::TransactionStatus;
-use frame_support::{
-	construct_runtime,
+pub use frame_support::{
```

Comment:
This should not be `pub`. This was recently corrected, and probably slipped back in here by accident in a conflict resolution.